controllers/gce/controller/utils.go

```diff
@@ -640,3 +662,23 @@ func (o PodsByCreationTimestamp) Less(i, j int) bool {
 	}
 	return o[i].CreationTimestamp.Before(o[j].CreationTimestamp)
 }
+
+func addInstanceGroupsAnnotation(existing map[string]string, igs []*compute.InstanceGroup) (map[string]string, error) {
+	if existing == nil {
+		existing = map[string]string{}
+	}
```

Comment:
I prefer asserting that the map exists outside of this func. Changing this would mean we don't need the first return variable either.

On another note, a less overloaded term than `add` would be nice, ie `set` or `apply`